Dockerfile

```diff
@@ -11,7 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM --platform=$BUILDPLATFORM golang:1.17.8 as builder-env
+FROM --platform=$BUILDPLATFORM ghcr.io/oracle/oraclelinux:8-slim as builder-env
```

Comment:
As I recall, you used 7-slim for the Rancher stuff, and 8-slim here. Curious why the difference.

Reply:
On discussion with other folks looks like we are beginning to move to oraclelinux8 for new stuff. I will change the rancher operator to oracle 8 as well.